misc/testerina/modules/testerina-core/src/main/ballerina/report.bal

```diff
@@ -91,10 +91,16 @@ function consoleReport(ReportData data) {
     }
 }
 
-function formatFailedError(string message) returns string {
+function formatFailedError(string message, int tabCount) returns string {
```

Comment:
is this function used with different tab counts? Or do you see any future usages?
If not, shall we make this tabCount a method-level variable within this `formatFailedError` function itself?

Reply:
I noticed another usage. So, we can keep this as it is.